#!/usr/bin/env python

PACKAGE = "mur_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("p_yaw", double_t,   0, "Proportional parameter Yaw",       5, 0, 20)
gen.add("d_yaw", double_t,   0, "Derivative parameter Yaw",         10.0, 0, 50)

gen.add("angle", double_t,  0, "Position Yaw desire",   2.0, -4, 4)

exit(gen.generate(PACKAGE, "mur_yaw_control", "MurYawControl"))
